src/sonic-bmpcfgd/scripts/bmpcfgd

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+'''
+bmpcfgd
+Daemon which monitors bmp relevant table enablement from CONFIG_DB, and reset BMP states
+'''
+
+import os
+import sys
+import subprocess
+import syslog
+import signal
+from shutil import copy2
+from datetime import datetime
+from sonic_py_common import device_info
+from sonic_py_common.general import check_output_pipe
+from swsscommon.swsscommon import ConfigDBConnector, DBConnector, Table
+from swsscommon import swsscommon
+
+CFG_DB = "CONFIG_DB"
+STATE_DB = "STATE_DB"
```

Comment:
>STATE_DB [](http://example.com/codeflow?start=0&length=8)

As offline discussed, please use a dedicated database for BMP datasets, proposing the database name is BMP_STATE_DB. you can pick one unused database index [0, 100). 100 is current maximal.
Also change the constant name.

Reply:
done, use "STATE_DB" as constant first, will change it into dedicated DB later together.

Reply:
Still applicable